pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -1239,59 +1256,92 @@ func findBridge(bridgeName string) (*netlink.Bridge, error) {
 }
 
 // XXX Need additional logic for IPv6 underlays.
-func getUlAddrs(ctx *zedrouterContext,
-	ifnum int, appNum int,
-	status *types.UnderlayNetworkStatus,
-	netInstStatus *types.NetworkInstanceStatus) (string, string, error) {
-
-	log.Functionf("getUlAddrs(%d/%d)\n", ifnum, appNum)
+func getUlAddrs(ctx *zedrouterContext, ifnum int,
```

Comment:
Since you wanted to change the order of the arguments to this function, how about also removing the unused ifnum argument. It is only used for logging and with the appID the logs have enough info.